docs/scos/dev/front-end-development/migration-guide-upgrade-to-webpack-v5.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Migration guide - Upgrade to Webpack v5
+description: Use the guide to update versions of the Webpack and related modules.
+last_updated: Jan 12, 2023
+template: module-migration-guide-template
+---
+
+## Upgrading Webpack from version 4.* to version 5.*
+
+This document provides instructions for upgrading Webpack to version 5 in your Spryker project.
+
+## Overview
+
+The Webpack v5 was released two years ago.
+To unblock future upgrades of other dependencies, we need to migrate as well.
+
+*Estimated migration time: 1h*
+
+## 1) Update dependencies
+
+1. In `package.json`, update the following dependencies to the new version:
+
+```json
+{
+    "clean-webpack-plugin": "~4.0.0",
+    "compression-webpack-plugin": "~10.0.0",
+    "copy-webpack-plugin": "~11.0.0",
+    "mini-css-extract-plugin": "~2.7.2",
+    "optimize-css-assets-webpack-plugin": "~6.0.1",
+    "postcss": "~8.4.20",
+    "terser-webpack-plugin": "~5.3.6",
+    "webpack": "~5.74.0",
+    "webpack-merge": "~5.8.0"
+}
+```
+
+- If Merchant Portal is used:
+
+```json
+{
+    "@types/webpack": "~5.28.0"
+}
+```
+
+2. Update and install package dependencies:
+
+```bash
+rm -rf node_modules
+npm install
+```
+   
```

Comment:
If this is a verification process (line 52), then I suppose we could add a verification block here
```suggestion

{% info_block warningBox "Verification" %}

```